src/Monocle/Effects.hs

```diff
@@ -535,15 +536,31 @@ httpRequest request = do
   HttpEffect manager <- getStaticRep
   unsafeEff_ $ HTTP.httpLbs request manager
 
+httpRequestWithTimeout ::
+  (HasCallStack, Exception e) =>
```

Comment:
Could not deduce ‘Exception e0’

Reply:
The type variable `e` is unused, try without the constraint.